frontend/src/test/scala/bloop/bsp/BspClientTest.scala

```diff
@@ -318,7 +318,7 @@ trait BspClientTest {
                 val report = diagnostics.map(
                   _.copy(source = Some("_"), code = Some("_")).toString
                     .replace("\n", " ")
-                    .replace(System.lineSeparator, " ")
+                    .replace("\n|\r\n|\r", " ")
```

Comment:
Best probably to make `END_OF_LINE_MATCHER` public and use it here.

Reply:
ok

Reply:
ok